pkgs/applications/display-managers/lightdm/run-dir.patch

```diff
@@ -6,7 +6,7 @@ index 16b80f7..b3af435 100644
  #guest-account-script=guest-account
  #logind-check-graphical=false
  #log-directory=/var/log/lightdm
--#run-directory=/var/run/lightdm
+-#run-directory=/run/lightdm
 +run-directory=/run/lightdm
```

Comment:
I already did this
https://github.com/NixOS/nixpkgs/blob/34a94e3b75ed59ee83c22394d5976933bd3c48ab/pkgs/applications/display-managers/lightdm/run-dir.patch#L10